### DISCUSSION TITLE
Parse all project files for errors

### DISCUSSION THREAD

**Discussion by AlmiAlmo** [microsoft/vscode-cpptools#8743] [Q&A]
Hello.

For now intellisense parse only currently opened and selected file for errors. Errors in closed or not selected files is never has been found. Can i enable this somehow?

**Comment by sean-mcmanus** [comment]
There is no way to enable this currently. Theoretically, it could be implemented in the future via a mechanism similar to what is used by the "Run Code Analysis on All Files" command.

**Reply by AlmiAlmo** [reply]
Ok. Ty for answer.
